### ISSUE TITLE
Error loading pyRevit

### ISSUE THREAD

**Issue by chttIE** [eirannejad/pyRevit#1949]
Good afternoon.
First of all I would like to thank you for pyrevit. thx!)
![image](https://github.com/eirannejad/pyRevit/assets/128715491/cc7fb052-7970-4ced-a5c1-dc877cf803f7)

The error shown on the screen last time was solved by changing the manifest name to 0Pyrevit.....
But I would like all plugins to just be friends with each other without crutches.

Digging deeper, it became clear that the library traces are a bit behind those used by other plugins, which is why the error appears.
![image](https://github.com/eirannejad/pyRevit/assets/128715491/c338e689-2032-463e-b4b3-960e913957d9)
especially ControlzEx.dll

Tell me if you are going to move to new versions of libraries

**Comment by jmcouffin** [comment]
Thanks for digging in.
Please create a PR to the develop branch with these updated dll version, and we will see how it goes once the CI-CD kicks in

**Comment by jmcouffin** [comment]
@chttIE 
- ControlZEX you are right
- Mahapps is recompiled under pyrevitlabs, so it should not be an issue
- MaterialDesign* is not part of the repo / project, or at least I cannot find it.

**Comment by jmcouffin** [comment]
https://pyrevitlabs.notion.site/Custom-Changes-d9ea2ad6b6174bb4b583e20ab08fb0ef

https://github.com/eirannejad/pyRevit/issues?q=is%3Aissue+MahApps+

**Comment by Wilionaire** [comment]
I have this same issue, followed most of the fixes (pyrevit env + pyrevit doctor in cmd) removed all the add-ins I could. Reinstalled etc etc. Still having the same issue. Appreciate all the hard work from everyone, Im not skilled enough to help. Just to put my hand up with the same issue,

**Comment by jmcouffin** [comment]
@Wilionaire try and prefix the pyRevit.addin file like so 0_pyRevit.addin and report if it works

**Comment by Wilionaire** [comment]
@jmcouffin thanks. In my previous attempts to fix the issue I had uninstalled and reinstalled pyrevit. The error was being presented, but then pyrevit would not load. 

After adding the prefix as suggested, I am still getting the error, but pyrevit is now available which is where we were at the start of this. I tested the prefix across multiple years 20, 21, 22 and 23. The error and pyrevit tab present in all except 22. In 22, the error presented by no pyrevit tab. I hope this helps?
